static/src/javascripts/projects/common/modules/gallery/lightbox.js

```diff
@@ -140,7 +146,7 @@ define([
             }, {
                 'text': 'Pinterest',
                 'css': 'pinterest',
-                'url': encodeURI('http://www.pinterest.com/pin/create/button/?description=' + config.page.webTitle + '&url=' + blockShortUrl + '&media=' + GalleryLightbox.prototype.getImgSrc(img, '700', '700'))
+                'url': encodeURI('http://www.pinterest.com/pin/create/button/?description=' + config.page.webTitle + '&url=' + blockShortUrl + '&media=' + urlPrefix + currentImage)
```

Comment:
undefined casts to 'undefined' no?

can define it at declaration time something like this:
```javascript
urlPrefix = /^\/\//.exec(currentImage) === '//' ? 'http:' : ''
```